src/binaryutils.cpp

```diff
@@ -324,6 +337,33 @@ InferenceEngine::Blob::Ptr convertMatsToBlob(std::vector<cv::Mat>& images, const
     }
 }
 
+ov::runtime::Tensor convertMatsToBlob_2(std::vector<cv::Mat>& images, const std::shared_ptr<TensorInfo>& tensorInfo, bool isPipeline) {
+    switch (tensorInfo->getPrecision()) {
+    case InferenceEngine::Precision::FP32:
+        return createBlobFromMats_2(images, tensorInfo, isPipeline);
```

Comment:
Why so many returns are needed? You could group all cases with "positive" returns.

Reply:
OV 1.0 version needed to be like that because it was templatized. Here we no longer need templates so you are right - we can group all cases.

Reply:
Fixed